next.config.js

```diff
@@ -9,10 +10,11 @@ const nextConfig = {
   swcMinify: true,
   images: {
     domains: ["mlog-lygggg.s3.ap-northeast-2.amazonaws.com"],
+    formats: ["image/webp"],
   },
 };
 
 module.exports = (_phase, { defaultConfig }) => {
-  const plugins = [withBundleAnalyzer];
+  const plugins = [withPlaiceholder, withBundleAnalyzer];
```

Comment:
```suggestion
  const plugins = [withPlaceholder, withBundleAnalyzer];
```
요 extension 설치해보세요.

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
저도 처음에 잘못친건가했는데, https://plaiceholder.co/plugins/next 실제로 저렇게 지은것같습니다..ㅎ